src/System.IdentityModel.Tokens/Utility.cs

```diff
@@ -65,5 +65,25 @@ internal static string SerializeAsSingleCommaDelimitedString(IEnumerable<string>
 
             return sb.ToString();
         }
+
+        public static bool IsHttps(string address)
+        {
+            if (string.IsNullOrEmpty(address))
+            {
+                return false;
+            }
+
+            return address.StartsWith("https");
```

Comment:
It will return true. I am considering creating a Uri from the input string and validating the uri scheme (using the other overload). Does that sound correct?